scripts/lib/sign-win32.ts

```diff
@@ -52,39 +52,47 @@ export async function sign(workDir: string) {
 
   // Sign individual files.  See https://github.com/electron-userland/electron-builder/issues/5968
   const unpackedDir = path.join(workDir, 'unpacked');
-  const internalDir = 'resources/resources/win32/internal';
-
-  // make privileged-service.exe available to the instller during signing
+  const resourcesRootDir = 'resources/resources/win32';
+  const internalDir = path.join(resourcesRootDir, 'internal');
+  const binDir = path.join(resourcesRootDir, 'bin');
+  const whiteList: Record<string, Array<string>> = {
+    '.':                ['Rancher Desktop.exe'],
+    [resourcesRootDir]: ['wsl-helper.exe'],
+    [internalDir]:      ['host-resolver.exe', 'privileged-service.exe', 'steve.exe', 'vtunnel.exe'],
+    [binDir]:           ['docker.exe', 'docker-credential-none.exe', 'kuberlr.exe', 'nerdctl.exe', 'rdctl.exe'],
```

Comment:
We should add a comment that `docker.exe` is built by us, and therefore we need to sign it.

Reply:
done